screens/WalletDetailScreen.tsx

```diff
@@ -10,13 +10,16 @@ import {
   ScrollView,
   StyleSheet,
   TouchableOpacity,
-  View
+  View,
+  Text
 } from "react-native";
 import { NavigationScreenProps } from "react-navigation";
 import BigNumber from "bignumber.js";
 
 import useBlockheight from "../hooks/useBlockheight";
 import useSimpleledgerFormat from "../hooks/useSimpleledgerFormat";
+import EyeOpenIcon from "../assets/images/eye-open.png";
+import EyeClosedIcon from "../assets/images/eye-closed.png";
```

Comment:
This app already has many icon libraries available to use.
For example on how to use them, see 
https://github.com/Bitcoin-com/badger-mobile/blob/develop/components/CoinRow/CoinRow.tsx#L5

Here's a page with all of the icons already available
https://oblador.github.io/react-native-vector-icons/

The eye icons imported here IMO are a bit unclear as to what the closed one means do, in the above page there are many more to choose from.  `FontAwesome5` and `Feather` both appear to have nice eyes with lines through icons.